src/iMatix.xml

```diff
@@ -0,0 +1,68 @@
+<SPDX name="iMatix Standard Function Library Agreement" identifier="iMatix" osi-approved="false">
+  <urls>
+    <url>http://legacy.imatix.com/html/sfl/sfl4.htm#license</url>
+  </urls>
+  <license>
+    <title>
+      <p>The SFL License Agreement</p>
+    </title>
+    <body>
+      <p>This license agreement covers your use of the iMatix STANDARD FUNCTION LIBRARY (SFL), its source code,
+         documentation, and executable files, hereinafter referred to as "the Product".</p>
+      <p>The Product is Copyright (c) 1991-2000 iMatix Corporation. You may use it and distribute it according to
+         this following License Agreement. If you do not agree with these terms, please remove the Product from
+         your system. By incorporating the Product in your work or distributing the Product to others you
+         implicitly agree to these license terms.</p>
+      <p>Statement Of Copyright</p>
+      <p>The Product is, and remains, Copyright © 1991-2000 iMatix Corporation, with exception of specific
+         copyrights as noted in the individual source files.</p>
+      <p>Conditions Of Use</p>
+      <p>You do not need to provide the source code for the Product as part of your product. However, you must do
```

Comment:
discussed on legal call; will add acknowledgement label to this PR so that we can make these conditional acknowledgement requirements extractable later